src/packet_analysis/Manager.h

```diff
@@ -112,6 +113,13 @@ class Manager : public plugin::ComponentManager<Tag, Component> {
 	 */
 	void ResetUnknownProtocolTimer(const std::string& analyzer, uint32_t protocol);
 
+	detail::PacketFilter* GetPacketFilter(bool init=true)
+		{
+		if ( ! pkt_filter && init )
+			pkt_filter = new detail::PacketFilter(detail::packet_filter_default);
```

Comment:
Is this cleaned up anywhere?

Reply:
Yes, in the `packet_analysis::Manager` destructor.